ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -77,8 +77,11 @@ public static boolean isWindowBound(final ColumnName columnName) {
     return WINDOW_BOUNDS_COLUMN_NAMES;
   }
 
+  public static boolean isPseudoColumn(final ColumnName columnName, final int pseudoColumnVersion) {
```

Comment:
Why is this needed? If it's just for testing purposes, we should mark it as such and it should also not be public.

Reply:
As mentioned above, I haven't fully convinced myself of where it'll end up being called, though given it's 1-arg overload is called quite a few places, I expect we'll be using it in some of those, especially after CREATE OR REPLACE is added. I'm, like before, assuming you have understanding I don't have on this, if that understanding is complex we can go over it in our next sync.

Reply:
As above, if the method doesn't need to be public (or if it doesn't even need to exist), then we should leave it out for now and introduce it later when it's actually needed, in order to avoid bloating the code.